app/src/main/res/values/strings.xml

```diff
@@ -55,6 +55,12 @@
     <string name="title_activity_home">Home</string>
     <string name="welcome">Welcome,</string>
 
+    <string name="display">Display</string>
```

Comment:
I think that when the card view is hidden, the button should be named "Show statistics". Otherwise, the user cannot guess what the button will open. Same for the french version.

Reply:
I thought about that and I think that the user can guess easily that it is about statistics because it is display by default when you arrive in the home page. Anyway, you are maybe right, we are never to careful regarding the guessing of a user.